### DISCUSSION TITLE
Can I embed non-public budibase app to another website?

### DISCUSSION THREAD

**Discussion by thanhpd56** [Budibase/budibase#13377] [Help] [answered]
Hi,
Does Budibase support to embed non-public app to another website (react js)? if user is not logged in then it will show login page.
Thanks.

**Comment by ConorWebb96** [comment, marked as answer]
Hey @thanhpd56,

Iframes only support public access to pages. Attempting to link an iframe to a non-public page will result in the error message: `This Budibase app is not publicly accessible`.

Hopefully this makes sense.
